src/CoreFx.Private.TestUtilities/src/System/PlatformDetection.NetFx.cs

```diff
@@ -2,64 +2,15 @@
-        // When we update xunit runner in the future which may target recent framework version, TargetsNetFx452OrLower can start return
-        // false but we don't expect any code change though.
-        public static bool TargetsNetFx452OrLower => TargetVersion.CompareTo(new Version(4, 5, 3, 0)) < 0;
-
-        public static bool IsNetfx462OrNewer => GetFrameworkVersion() >= new Version(4, 6, 2);
-
-        public static bool IsNetfx470OrNewer => GetFrameworkVersion() >= new Version(4, 7, 0);
-
-        public static bool IsNetfx471OrNewer => GetFrameworkVersion() >= new Version(4, 7, 1);
-
-        public static bool IsNetfx472OrNewer => GetFrameworkVersion() >= new Version(4, 7, 2);
-
-        // To get the framework version we can do it throught the registry key and getting the Release value under the .NET Framework key.
-        // the mapping to each version can be found in: https://docs.microsoft.com/en-us/dotnet/framework/migration-guide/how-to-determine-which-versions-are-installed
-        // everytime we ship a new version this method should be updated to include the new framework version.
-        private static Version GetFrameworkVersion()
-        {
-            using (RegistryKey ndpKey = Registry.LocalMachine.OpenSubKey(@"SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full"))
-            {
-                if (ndpKey != null)
-                {
-                    int value = (int)(ndpKey.GetValue("Release") ?? 0);
-                    if (value >= 461808)
-                        return new Version(4, 7, 2);
-                    if (value >= 461308)
-                        return new Version(4, 7, 1);
-                    if (value >= 460798)
-                        return new Version(4, 7, 0);
-                    if (value >= 394802)
-                        return new Version(4, 6, 2);
-                    if (value >= 394254)
-                        return new Version(4, 6, 1);
-                    if (value >= 393295)
-                        return new Version(4, 6, 0);
-                    if (value >= 379893)
-                        return new Version(4, 5, 2);
-                    if (value >= 378675)
-                        return new Version(4, 5, 1);
-                    if (value >= 378389)
-                        return new Version(4, 5, 0);
-
-                    throw new NotSupportedException($"No 4.5 or later framework version detected, framework key value: {value}");
-                }
-
-                throw new NotSupportedException(@"No registry key found under 'SOFTWARE\Microsoft\NET Framework Setup\NDP\v4\Full' to determine running framework version");
-            }
-        }
+        public static bool IsInContainer => false;
+        public static bool IsWindowsNanoServer => false;
+        public static bool IsWindowsServerCore => false;
+        public static bool SupportsSsl3 => false;
```

Comment:
@wfurt @davidsh do we test ssl3 on netfx in corefx currently? If yes then I need to make some changes here.